### PR TITLE
Translator plugin

### PR DESCRIPTION
**Pull request by a-deeb**
# brief description of changes
Added a Google Translate plugin but its missing an API number

### CODE REVIEW

File: honeybot/plugins/translate.py [hunk 1]
```diff
@@ -0,0 +1,340 @@
+# -*- coding: utf-8 -*-
+
+"""
+[weather.py]
```
Comment (Abdur-rahmaanJ) [inline honeybot/plugins/translate.py]: translate.py?

File: honeybot/plugins/translate.py [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+Turkish 	tr
+Ukrainian 	uk
+Vietnamese 	vi
+Welsh 	cy
+Yiddish 	yi
+"""
+import os
+import urlparse
+import urllib
+import urllib2
```
Comment (Abdur-rahmaanJ) [inline honeybot/plugins/translate.py]: woops this is a py3 bot, could you change please? if not within 3 days i'll merge it then change later

File: honeybot/plugins/translate.py (hunk 1 above)
Comment (a-deeb) [reply honeybot/plugins/translate.py]: Hi @Abdur-rahmaanJ , sorry about that i just made the changes now

File: honeybot/plugins/translate.py (hunk 2 above)
Comment (a-deeb) [reply honeybot/plugins/translate.py]: I just made these changed the py imports now , is this how they should be?

**Pull request merged**